mkdocs.yml

```diff
@@ -80,7 +80,7 @@ nav:
     - Analise and test: 'configure_service/analise_test.md'
   - Advanced reference:
     - 'advanced_reference/index.md'
-    - publish fetch: 'guides/publish_fetch_packages.md'
+    # - publish fetch: 'guides/publish_fetch_packages.md'
```

Comment:
This section is not revised yet, and most of the info is in the tutorial guide. Will put it back once it is thoroughly revised